.github/workflows/ci.yml

```diff
@@ -1,10 +1,17 @@
 name: ci
 
-on: [push, repository_dispatch]
+on:
+  pull_request_target:
+    types: [labeled]
+  push:
+    branches-ignore:
+      - "dependabot/**"
+  repository_dispatch:
 
 jobs:
   main:
     runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'dependencies') || github.event_name == 'push'
```

Comment:
Will this make it not run if it's a repository_dispatch event?

Reply:
looks like it! i'll update this.